check.py

```diff
@@ -175,6 +175,31 @@ def do_asm2wasm_test():
               fail_with_error('wasm interpreter error: ' + err) # failed to pretty-print
             fail_with_error('wasm interpreter error')
 
+        # verify debug info
+        if 'debugInfo' in asm:
+          jsmap = 'a.wasm.map'
+          cmd += ['--binarymap-file', jsmap,
```

Comment:
so `--binarymap-file` emits a source map? what i am asking is, should we change the parameter names to `--source-map` or something similar, if they just emit a source map?

or am I missing something here? is there still a binary file, with a "reference"? if so where is that documented, i don't think i see it in t he design pr?